src/components/reusable/Tabs.js

```diff
@@ -19,10 +19,17 @@ function Tabs({ options, selected, setSelected }) {
 
 const StyledTabs = styled.div`
   .tab {
-    border: 1px solid black;
+   margin-top:3rem;
```

Comment:
you have 3 different values for margin here, making two of them redundant. Maybe decide on the one that works best.